src/components/formPages/login/LoginPage.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import { GoogleButtonUI, LogInButtonUI } from '../colorButton';
+import GoogleLogo from '../logos/GoogleLogo';
+import { Link } from 'react-router-dom';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { useForm } from 'react-hook-form';
+import { useState } from 'react';
+import { login } from '../../axios/axiosForms';
+import { MoonLoginImage } from '../../../assets/images/Moon';
+import { regex } from '../../../assets/variables';
+import {
+  LoginScreenStyle,
+  DisplayScreen,
+  SubmitDisplay,
+  StyleDivider,
+} from '../../../assets/styles/login.style';
+import { useNavigate } from 'react-router-dom';
+
+const LoginPage = () => {
+  const [email, setEmail] = useState<string>('');
+  const [password, setPassword] = useState<string>('');
+  const history = useNavigate();
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
```

Comment:
If we are using react hook form please follow correct pattern for that (handlesubmit must be passed to form)